oidc-client-app-samples/browser-client-app-sample/app/index.html

```diff
@@ -0,0 +1,114 @@
+~ limitations under the License.
+-->
+<html>
+
+<head>
+  <title>WSO2-IAM OIDC App</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body class="login-dispatch">
+  <section id="login">
+    <div class="container">
+      <div class="row">
+	<div class="col-md-4 col-md-offset-4">
+          <div class="form-wrap-header">
+	    <img src="img/logo.png" class="center-block pickup-logo-login" alt="pickup-logo" />
+            <span class="app-title center-block">WSO2-IAM OIDC Public Client</span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+          <div class="form-wrap">
+            <!--button class="btn btn-custom btn-lg btn-block custom-primary add-margin-top-4x" id="fetchConfig">Fetch Configuration</button-->
+            <button class="btn btn-custom btn-lg btn-block custom-primary add-margin-top-4x" id="authorize">Login</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer id="footer" class="login-footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <span class="text-muted">Copyright &copy;  <a href="http://wso2.com/" target="_blank" class="wso2-logo-link"><img src="img/wso2-light.svg" class="wso2-logo" alt="wso2-logo" /></a> &nbsp;2018</span>
+        </div>
+      </div>
+    </div>
+  </footer>
+
+  <!-- extras -->
+  <!-- Bootstrap Core CSS -->
+  <link href="libs/bootstrap_3.3.7/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" href="./css/app.css" />
+
+  <!-- Custom Fonts -->
+  <link href="libs/font-awesome_4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+  <link href="css/font.css" rel="stylesheet" type="text/css">
+  <!-- scripts -->
+  <script src="js/jquery-3.2.1.min.js"></script>
+  <!-- Bootstrap Core JavaScript -->
+  <script src="libs/bootstrap_3.3.7/js/bootstrap.min.js"></script>
+  <script src="./js/bundle.js"></script>
+  <script type="text/javascript">
+
+    var application = null;
+    var init = function () {
+      application = new App({
+    'authorizeUrl': 'https://localhost:9443/oauth2/authorize',
+    'tokenUrl': 'https://localhost:9443/oauth2/token',
+    'revokeUrl': 'https://localhost:9443/oauth2/revoke',
+    'logoutUrl': 'https://localhost:9443/oidc/logout',
+    'userInfoUrl': 'https://localhost:9443/oauth2/userinfo',
+    'flowType': "PKCE",
+    'userStore': "LOCAL_STORAGE",
+    'clientId': '0Ob7gGlPWWoCMx4qrGNSWjRgnMwa',
+    'clientSecret': '4jjZU5nke4iovlq5seaSPVxnqBMa',
+    'redirectUri': 'http://localhost:8080/app/home.html',
+    'scope': 'openid',
+    'postLogoutRedirectUri': 'http://localhost:8080/app/'
+  });
+  application.init();
+      // check for authorization response if available.
+      application.checkForAuthorizationResponse();
+
+      // fetch configuration handler
```

Comment:
Describe when we have to uncomment this section in a comment

Reply:
done